### PR TITLE
Feature/flooding options api

### PR DESCRIPTION
**Pull request by Adamws33**
Updated flooding frequency to filter crops and pull options from API

### CODE REVIEW

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://172.210.117.121:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://172.210.117.121:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://172.210.117.121:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://172.210.117.121:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

File: src/pages/CropSidebar/CropSidebar.js [hunk 1]
```diff
@@ -155,11 +169,13 @@ const CropSidebar = ({
         }
       });
       cd[n].inactive = (!match)
-      || (drainageClassRedux && !crop.soilDrainage?.includes(drainageClassRedux));
+      || (drainageClassRedux && !crop.soilDrainage?.includes(drainageClassRedux))
```
Comment (mikahpinegar) [inline src/pages/CropSidebar/CropSidebar.js:172]: can you add in the .toLowercase() that we did for main here?

File: src/pages/CropSidebar/CropSidebar.js [hunk 2]
```diff
@@ -140,6 +140,20 @@ const CropSidebar = ({
     });
 
     const filtered = cropData?.filter((crop, n, cd) => {
+      const floodingFrequencyValue = crop.attributes.filter((a) => a.label === 'Flooding Frequency Tolerance')[0]?.values[0];
+      const getFloodingFrequency = () => {
```
Comment (mikahpinegar) [inline src/pages/CropSidebar/CropSidebar.js:144]: this function is confusing. i think i get what it's doing (returning the index of the highest drainage value associated with the string value) but it's not super obvious. can you either add comments to explain the logic and name the function more clearly or use a data structure that is more intuitive?

File: src/pages/CropSidebar/CropSidebar.js (hunk 1 above)
Comment (mikahpinegar) [reply src/pages/CropSidebar/CropSidebar.js:172]: if you merge in main it'll do it automatically

File: src/shared/constants.js [hunk 3]
```diff
@@ -918,7 +810,7 @@ export const cropDataFormatter = (cropData = [{}], cashCropStartDate = '', cashC
     });
     return vals;
   };
-
+  console.log(cropData);
```
Comment (mikahpinegar) [inline src/shared/constants.js:813]: remove clg

**Comment by Adamws33** [comment]
> choosing MCCC and no option for flooding filters out all crops

I was unable to replicate this error.  can you tell me the state and region you selected?

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://172.210.117.121:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://172.210.117.121:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://172.210.117.121:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://,172.210.117.121,:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://172.210.117.121:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://172.210.117.121:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Review by mikahpinegar — APPROVED** [review]

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://172.210.117.121:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://172.210.117.121:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Pull request merged**